src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -29,8 +30,17 @@
 
 import org.apache.cassandra.locator.IEndpointSnitch;
 import org.apache.cassandra.locator.InetAddressAndPort;
+import org.apache.cassandra.security.ISslContextFactory;
 import org.apache.cassandra.security.SSLFactory;
+import org.apache.cassandra.utils.FBUtilities;
 
+/**
```

Comment:
would you mind to make this Javadoc "nicer"? It is not formatted correctly, I mean, empty line at the end, then the last sentence is short and might be put on the same same line as "require" and so on ...